db/collection.go

```diff
@@ -263,6 +266,68 @@ func (c *Collection) Find(q *Query, opts ...TxnOption) (instances [][]byte, err
 	return
 }
 
+type filter struct {
+	Collection string
+	Time       int
+}
+
+func (f filter) Filter(e query.Entry) bool {
+	// Easy out, are we in the right collection?
+	key := ds.NewKey(e.Key)
+	base := key.Parent().BaseNamespace()
+	number, _ := strconv.Atoi(base)
+	kind := key.Type()
+	return kind == f.Collection && number > f.Time
+
+}
+
+// ModifiedSince returns a list of all instances that have been modified (and/or touched) since `time`.
+func (c *Collection) ModifiedSince(time int64, opts ...TxnOption) (modified []core.InstanceID, err error) {
```

Comment:
Note that I have _not_ yet exposed this via any gRPC APIs/updates. Partly because I can work around having this for now using less efficient means, and partly because it should be reviewed before I bother exposing it.

Reply:
Would it make sense to combine this with `Find`, possibly as a new kind of query / condition? Then you could combine this functionality with other queries / conditions.

Reply:
Hmmm, interesting. That would be ideal (no new APIs). I'm not sure off the top of my head how this might work given we are querying the dispatcher directly... but as you suggest, some sort of additional query condition/operator might do the trick. Will look into this. In that case, I might mark that API as experimental, not expose it via gRPC for now, and explore this alternative in a follow on PR. Seem reasonable?

Reply:
SGTM!